doc/source/whatsnew/v1.3.0.rst

```diff
@@ -294,6 +294,7 @@ Indexing
 - Bug in setting ``timedelta64`` values into numeric :class:`Series` failing to cast to object dtype (:issue:`39086`)
 - Bug in setting :class:`Interval` values into a :class:`Series` or :class:`DataFrame` with mismatched :class:`IntervalDtype` incorrectly casting the new values to the existing dtype (:issue:`39120`)
 - Bug in incorrectly raising in :meth:`Index.insert`, when setting a new column that cannot be held in the existing ``frame.columns``, or in :meth:`Series.reset_index` or :meth:`DataFrame.reset_index` instead of casting to a compatible dtype (:issue:`39068`)
+- Bug in :meth:`RangeIndex._concat` where a single object of length 1 was concatenated incorrectly (:issue:`39401`)
```

Comment:
is there a user-facing method that we can reference instead of RangeIndex._concat?

Reply:
RangeIndex.append method is making use of RangeIndex._concat. We can reference it instead.